pymc_experimental/tests/test_model_builder.py

```diff
@@ -237,3 +237,12 @@ def test_id():
     ).hexdigest()[:16]
 
     assert model_builder.id == expected_id
+
+
+def test_step_selection_in_sample_config(toy_X, toy_y):
+    sampler_config = {
+        "step": "Slice",
+    }
+    model = test_ModelBuilder(sampler_config=sampler_config)
+    model.fit(toy_X, toy_y)
```

Comment:
Maybe monkeypatch `pm.sample` to confirm it was called with `step` and that's `instance(step, Slice)`?